### ISSUE TITLE
[Bug]Disabling uBlock and quickly opening the Settings section crashes Fenix

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8570]
## Prerequisites
Have uBlock Origins installed.

## Steps to reproduce
1. Disable uBlock and quickly tap the Settings section.

### Expected behavior
The Settings section should be properly loaded.

### Actual behavior
Crash

### Device information

* Android device: 
•  LG G7 fit (Android 8.1)
•  Huawei P9 Lite (Android 7)
• OnePlus A3 (Android 6.0.1)

* Fenix version: 
• Beta 4.0.0-beta.1 from 2/19
• Nightly #20510607 from 2/20
• Beta Migration build 2/19
• Nightly migration build 9, 02/14

### Notes
► Log
```kotlin
    --------- beginning of crash
02-20 12:26:43.460 28685-28685/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fenix.nightly, PID: 28685
    org.mozilla.geckoview.GeckoResult$UncaughtException: java.lang.IllegalStateException: remove_add_on must not be null
        at org.mozilla.geckoview.GeckoResult.dispatchLocked(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.completeExceptionally(GeckoResult.java:4)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:10)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda)
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:152)
        at android.app.ActivityThread.main(ActivityThread.java:5507)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.IllegalStateException: remove_add_on must not be null
        at -$$LambdaGroup$ks$3dk2TV2PDS17-TeNsA6Slr1xVgA.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:8)
        at -$$LambdaGroup$ks$O1hItAye9K862nmb-1vtC5PRRfE.invoke(com.android.tools.r8.jetbrains.kotlin-style lambda group:42)
        at -$$LambdaGroup$js$8vC4ZcgGL9QCmG3TDOYl97vbFvI.onValue(java-style lambda group:52)
        at org.mozilla.geckoview.GeckoResult.lambda$thenInternal$2$GeckoResult(GeckoResult.java:2)
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$gwCgOUK_EYQn2g6GolfZvo6A_WE.run(lambda) 
        at org.mozilla.geckoview.GeckoResult.lambda$dispatchLocked$3(GeckoResult.java:2) 
        at org.mozilla.geckoview.-$$Lambda$GeckoResult$PehjG2jgyDa_p37vZrKrSi2I94s.run(lambda) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:152) 
        at android.app.ActivityThread.main(ActivityThread.java:5507) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
02-20 12:26:43.460 28685-28685/? D/AppTracker: App Event: crash
```
► Video
![20200220-122232](https://user-images.githubusercontent.com/51314259/74925185-6f185e80-53dc-11ea-8a3f-fab97cd3339e.gif)

**Comment by Amejia481** [comment]
@AndiAJ this should be fixed by now, would you mind taking a look at it?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20520607 from 2/21`** using the following devices:

• Google Pixel 3a XL (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

► Video from OnePlus A3
![20200221-172046](https://user-images.githubusercontent.com/51314259/75047595-b3d0f200-54cf-11ea-89e0-bce1d1868bab.gif)
